docs/includes/quickstart-choose-editor.md

```diff
@@ -1 +1 @@
-You can create an Office Add-in by using Visual Studio or any other editor. Tell us what editor you'd like to use by choosing one of the following tabs:
+You can create an Office Add-in by using the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office) or Visual Studio. The Yeoman generator creates a Node.js project that can be managed with Visual Studio Code or any other editor, whereas Visual Studio creates a Visual Studio solution. Tell us which one you'd like to use by choosing one of the following tabs.
```

Comment:
I'm not sure how I feel about "tell us". It sounds like we're collecting data or requiring an active input. Maybe some more passive like "Select your preferred Add-in creation method by choosing one of the following tabs."

Reply:
Agreed -- I never have liked that verbiage either. I've updated the sentence to this instead:  

Select the tab for the one you'd like to use and then follow the instructions to create your add-in and test it locally.